packages/components/src/components/dropdown/dropdown.ts

```diff
@@ -57,6 +53,16 @@ export default class SdDropdown extends SolidElement {
    */
   @property({ type: Boolean, reflect: true }) open = false;
 
+  /**
+   * Indicates whether or not the dropdown should be styled with rounded corners.
+   */
+  @property({ type: Boolean, reflect: true }) rounded = false;
+
+  /**
+   * Indicates whether or not the dropdown should automatically resize its content's width/height regarding the available space on screen.
+   */
+  @property({ type: Boolean, reflect: true, attribute: 'no-auto-size' }) noAutoSize = false;
+
   /**
    * The preferred placement of the dropdown panel. Note that the actual placement may vary as needed to keep the panel
    * inside of the viewport.
```

Comment:
Regarding the placement attribute: Should it be possible to leave this empty? Because currently I can select an empty string in storybook. Not sure, just asking.

Reply:
If you do, the default will be used. So it's fine.